README.md

```diff
@@ -94,10 +94,16 @@ You configure your lambda here.
     }
     # You can put any HCL-Map with valid Functionbeat config for Elasticsearch Output 
     output_elasticsearch = {
-      hosts : ["https://your-endpoint:443"]
-      protocol : "https"
-      username : "elastic"
-      password : "mysupersecret"
+      hosts = ["https://your-endpoint:443"]
+      protocol = "https"
+      username = "elastic"
+      password = "mysupersecret"
+    }
+
+    # You can also output to logstash, if you like
```

Comment:
To keep it clean i'd like to have this in a seperate section or a hint that you can use either output_elasticsearch or output_logstash. Maybe it's worth to even have a bullet point in the README for the different outputs, that mentions that exactly one output is required.